src/Flow/ETL/Transformer/EntryValueTransformer.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Transformer;
+
+use Flow\ETL\Exception\RuntimeException;
+use Flow\ETL\Row;
+use Flow\ETL\Row\EntryFactory;
+use Flow\ETL\Row\Factory\NativeEntryFactory;
+use Flow\ETL\Rows;
+use Flow\ETL\Transformer;
+
+/**
+ * @psalm-immutable
+ */
+final class EntryValueTransformer implements Transformer
```

Comment:
CallUserFunctionTransformer should better reflect responsibility of this transformer, wdyt?